packages/frontend/src/components/accounts/SetPassword/lib/validation.ts

```diff
@@ -0,0 +1,7 @@
+const MIN_PASS_LEN = 8;
+const MAX_PASS_LEN = 20;
```

Comment:
Is this a completed limitation? Somebody (for example me) would be uncomfortable with such max password length )